source/developer/webhooks-incoming.rst

```diff
@@ -160,7 +160,7 @@ Some common error messages include:
 
 If your integration prints the JSON payload data instead of rendering the generated message, make sure your integration is returning the ``application/json`` content-type.
 
-For further assistance, review the `Troubleshooting forum <https://forum.mattermost.org/c/trouble-shoot>`__ for previously reported errors, or `join the Mattermost user community for troubleshooting help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
+For further assistance, review the `Troubleshooting forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`__ for previously reported errors, or `join the Mattermost user community for troubleshooting help <https://mattermost.com/ask-mattermost-community/>`_.
```

Comment:
```suggestion
For further assistance, review the `Troubleshooting forum <https://forum.mattermost.org/t/how-to-use-the-troubleshooting-forum/150>`__ for previously reported errors, or `join the Mattermost user community for troubleshooting help <https://mattermost.com/pl/default-ask-mattermost-community/>`_.
```

Reply:
I suspect this link has tracking on it, hence the structure of the URL. @jasonblais, is this correct or should we change it?

Reply:
But it leads to a similar page I think?

Reply:
`pl` refers to a permalink. It has no tracking but it is used as the default page for Mattermost troubleshooting help, and should not be changed.

The reason we use https://mattermost.com/pl/default-ask-mattermost-community/ everywhere is so that if the "ask the community" webpage moves, we only have to change the redirect for https://mattermost.com/pl/default-ask-mattermost-community/ and avoid breaking all pages where the trourbleshooting help is referenced (handbook, docs, website, product).

Some more background about redirects here: https://handbook.mattermost.com/operations/research-and-development/product/product-management-team-handbook#how-do-i-create-redirects-for-in-product-documentation

Reply:
Let me know if that helps clarify it :)

Reply:
Okay got it. Thank you @jasonblais